monochart/templates/statefulset.yaml

```diff
@@ -99,6 +99,12 @@ spec:
         - mountPath: {{ .Values.statefulset.persistence.mountPath | quote }}
           name: {{ include "common.fullname" . }}
         {{- end }}
+        {{- if .Values.FirstVolumeMounts }}
+{{ tpl .Values.FirstVolumeMounts . | indent 8 }}
+        {{- end }}
+        {{- if .Values.extraVolumeMounts }}
+{{ tpl .Values.extraVolumeMounts . | indent 8 }}
+        {{- end }}
```

Comment:
I suppose that it could be done as 1 config, but if that move us to use original chart, I'm OK with it

Reply:
I believe there are two separate blocks for columes because of initContainers that could need a volume mounted. To make this change exactly same as in deployments, I probably should add it to initContainers too.
[This is how it looks like:](https://github.com/SpotOnInc/helmcharts/blob/master/monochart/templates/deployment.yaml#L87-L93)

```yaml
        {{- if .Values.FirstVolumeMounts }}
        volumeMounts:
{{ tpl .Values.FirstVolumeMounts . | indent 8 }}
        {{- if .Values.extraVolumeMounts }}
{{ tpl .Values.extraVolumeMounts . | indent 8 }}
        {{- end }}
        {{- end }}
```

Reply:
No idea if this has to be that way in order to work. Note that `volumeMounts:` for initContainers is added only if there is something in `FirstVolumeMounts`.